src/assets/less/common.less

```diff
@@ -106,28 +106,64 @@ img {
 
 .modal .tab-content {
   height: 100%;
-  max-height: calc(100vh - 300px);
-  overflow-y: scroll;
+  max-height: 98%;
+  overflow-y: auto;
 }
 
 #settings-modal .modal-content {
-  height: 671px;
+  height: 90vh;
+  max-height: 90vh;
 }
 
 .modal .settings-group {
-  text-align: center;
-  max-width: 250px;
-  margin-left: auto;
-  margin-right: auto;
+    text-align: center;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    margin-left: auto;
+    margin-right: auto;
 }
-.modal .settings-group > div {
-  float: left;
-  clear: both;
+
+
+div.custom-control {
```

Comment:
Could you make the name more meaningful than custom-control?

Reply:
custom-control class name is already being used in the project before I started working on it. So instead of creating a new CSS class, I decided to re-use it.